### ISSUE TITLE
Provide status announcements for interactions involving page loads

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1089]
There are numerous instances in the Omeka admin where the user can interact with the UI and encounters a page load. Examples include sort links, filtering browse views, deleting tags, and many others. After the page loads, Omeka needs to provide screen readers with a status announcement so that users can understand whether the interaction has succeeded or failed. 

Previously I've experimented with injecting status alerts on page load dynamically, but results have been inconsistent. Accessibility audits have recommended adding short status messages to `<title>` attributes, something like "Sorting by date (ascending) - Browse Items (454 results) - Site Name - Omeka Admin". Based on conversations in omeka/plugin-ActivityLog#1, that seems difficult without changing the page's `<h1>`/page title.

**Comment by kimisgold** [comment]
That thread feels pretty validating for avoiding aria approaches on page load. It looks like the issue is with screen reader implementation, and that lines up with my experience trying to use `aria-live` for status announcements with Voiceover a couple years ago. We could do some simple testing to see if that's still the case before getting into an approach using the `<title>`.

**Comment by aviannamiller** [comment]
I did some quick testing, and iOS and Android are pretty consistent at announcing `aria-live` regions / alerts on page load. Both JAWS and NVDA seem to handle these announcements as well for Edge and Chrome, but not Firefox, and I can't necessarily speak to the overall consistency there. So, yes, it appears the main issue here is with VoiceOver (on desktop), which does not make the announcement on page load on any major browser.

**Comment by kimisgold** [comment]
There's now a branch ([`a11y/page-load-sr-alerts`](https://github.com/omeka/Omeka/tree/a11y/page-load-sr-alerts)) with a draft of screen reader alerts for status changes involving a page load. Currently if a page loads with a status alert, like a success or error message, it loads the content of those alerts into an `aria-live` region a couple seconds after page load. If this works, I'll figure out how to apply the pattern to things like filtered or sorted browse views.
